pom.xml

```diff
@@ -3,7 +3,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>software.amazon.cloudformation</groupId>
     <artifactId>aws-cloudformation-rpdk-java-plugin</artifactId>
-    <version>1.0.4</version>
```

Comment:
1.0.4 wasn't released, just errantly put as the version number in the previous PR